_includes/header.html

```diff
@@ -42,7 +42,7 @@
         <input type="hidden" name="affiliate" id="affiliate" value="login.gov" />
         <div role="search">
           <input type="search" name="query" id="query" autocomplete="off" class="usa-input" />
-          <input type="submit" class="usa-button app-search" name="commit" value=" " data-disable-with="Search" />
+          <input type="submit" class="usa-button app-search" name="commit" value="Search" data-disable-with="Search" />
```

Comment:
translate the `value="Search"` here?

Reply:
Uggggggggh....good catch again!  Thank you.